dashboard/test/controllers/level_starter_assets_controller_test.rb

```diff
@@ -50,6 +50,40 @@ class LevelStarterAssetsControllerTest < ActionController::TestCase
     assert_equal file_objs[1].size, starter_assets[1]['size']
   end
 
+  test 'show: returns template level starter_assets when defined' do
+    uuid_name_1 = "#{SecureRandom.uuid}.png"
+    key_1 = "starter_assets/#{uuid_name_1}"
+    uuid_name_2 = "#{SecureRandom.uuid}.jpg"
+    key_2 = "starter_assets/#{uuid_name_2}"
+    file_objs = [
+      MockS3ObjectSummary.new(key_1, 123, 1.day.ago),
+      MockS3ObjectSummary.new(key_2, 321, 2.days.ago)
```

Comment:
Do you need this second object?

Reply:
We probably don't technically need to mock it for S3 access, but we should seed the child level with a starter asset to ensure it gets overwritten by the parent level.

Reply:
Yeah I meant the S3 mock, fine to leave as-is...just wanted to make sure I understood.

Reply:
Fixed!